src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int GROWTH_FACTOR = 2;
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default for int